src/CodeGenerators/SettingsGen/Models/ParameterModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Microsoft.Omex.CodeGenerators.SettingsGen.Models
+{
+	/// <summary>
+	/// Parameter element used for service fabric settings
+	/// </summary>
+	public sealed record ParameterModel : IEqualityComparer<ParameterModel>, IEquatable<ParameterModel>
```

Comment:
The point for using records is that you don't need to make equality manully and properties

Reply:
For example https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQMwAIBOBTAxge02HQAUBDTMgW2wBdtMBZfYbAGwAoo4AGdAOWrYANOm58AamTYBXEegBG+fG3SMZAZ1oB5AG4NMAS1YBKANxA=

Reply:
Had issues with equals in section settingsxml models